src/Npgsql/NpgsqlTypes/NpgsqlTsVector.cs

```diff
@@ -10,7 +10,7 @@ namespace NpgsqlTypes;
 /// <summary>
 /// Represents a PostgreSQL tsvector.
 /// </summary>
-public sealed class NpgsqlTsVector : IEnumerable<NpgsqlTsVector.Lexeme>
+public sealed class NpgsqlTsVector : IEnumerable<NpgsqlTsVector.Lexeme>, IEquatable<NpgsqlTsVector>
```

Comment:
Aren't all this changes technically 'breaking' and might lead to some issues if you attempt to replace the dll without recompiling?

Reply:
Do you mean you think there are binary breaks here? If so, where?

Aside from that, the changes seem like pretty straightforward bug fixes - it's always possible for a bug fix to break someone relying on the buggy behavior, but any particular danger/risk here?

Reply:
I always thought that adding an interface (or overriding methods) can be potentially problematic. If not, well, that's good to know)

Reply:
I'm not sure this case is problematic, considering that the class is sealed and the implementation for that interface is provided in the PR... I may be missing something, but I really doubt the case above is problematic.